package.json

```diff
@@ -30,10 +29,10 @@
   "scripts": {
     "lint": "eslint -c ./.eslintrc features/step_definitions/ features/support/",
     "test": "npm run lint && node ./node_modules/cucumber/bin/cucumber.js features/ -p verify && node ./node_modules/cucumber/bin/cucumber.js features/ -p mld",
+    "nodejs-tests": "make -C test/data && ./lib/binding/osrm-datastore test/data/ch/monaco.osrm && node test/nodejs/index.js | faucet",
```

Comment:
Formats the machine readable tap format output tape generates for humans, much easier to locate failing tests and see why they are failing. Not sure if faucet is the right pkg for this, I just found it and used it. We can change this.